### PR TITLE
Kma 21 navigation

### PR DESCRIPTION
**Pull request by m-lo**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
```
Comment (ericgmoon) [inline App.js:1]: This file shouldn't have been committed.

File: package.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
     "eslint-config-prettier": "^7.2.0",
     "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-prettier": "^3.3.1",
-    "expo": "~40.0.0",
-    "expo-font": "~8.4.0",
+    "expo": "^1.0.0",
```
Comment (ericgmoon) [inline package.json:20]: Why is this necessary?

File: src/packages/common/chip/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    >
+      <Button
+        value={value}
+        buttonStyle={styles.button}
+        titleStyle={{
+          ...styles.buttonTitle,
+          color: active ? "white" : "black",
+        }}
+        iconRight
+        {...rest}
```
Comment (ericgmoon) [inline src/packages/common/chip/index.js:20]: Could we do `...others` for consistency? I believe this is how the official React docs do it as well - it would be more recognisable.

File: src/packages/navigation/constants.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export default {
```
Comment (ericgmoon) [inline src/packages/navigation/constants.js:1]: Is this consts file really necessary? I can only see these constants being used once - and the original names aren't particularly confusing.

File: src/packages/navigation/index.js [hunk 5]
```diff
@@ -1,3 +1,24 @@
...
+  current,
+  direction = Direction.HORIZONTAL,
+}) {
+  return (
+    <View style={{ display: "flex", flexDirection: direction, padding: 8 }}>
+      {items.map((item) => (
+        <Chip
+          label={item.label}
+          value={item.value}
+          style={{ margin: 4 }}
```
Comment (ericgmoon) [inline src/packages/navigation/index.js:18]: Is there a reason why this wasn't moved to a `styles.js`?

File: src/packages/navigation/index.js [hunk 6]
```diff
@@ -1,3 +1,24 @@
...
+
+import Chip from "../common/chip";
+
+export default function Navigation({
+  items,
+  current,
+  direction = Direction.HORIZONTAL,
+}) {
+  return (
+    <View style={{ display: "flex", flexDirection: direction, padding: 8 }}>
```
Comment (ericgmoon) [inline src/packages/navigation/index.js:13]: Is there a reason why this wasn't moved to a styles.js?

**Comment by ericgmoon** [comment]
Could we also see a screenshot of the working component? The code itself looks pretty good :+1:

File: package.json (hunk 2 above)
Comment (m-lo) [reply package.json:20]: ummm its npm doing things not me...

File: src/packages/common/chip/index.js (hunk 3 above)
Comment (m-lo) [reply src/packages/common/chip/index.js:20]: kk

File: src/packages/navigation/constants.js (hunk 4 above)
Comment (m-lo) [reply src/packages/navigation/constants.js:1]: ok

File: src/packages/navigation/index.js (hunk 6 above)
Comment (m-lo) [reply src/packages/navigation/index.js:13]: ok

File: package.json (hunk 2 above)
Comment (ericgmoon) [reply package.json:20]: Hmm maybe undo this change :thinking: Not sure why this **significant downgrade** would have been necessary for the change.

**Comment by ericgmoon** [comment]
The change should also ideally come with appropriate tests, but since we're trying to get storybooks going for this ASAP, I can write those tests for you after this PR is merged in. How does that sound @m-lo?

**Comment by m-lo** [comment]
image

**Comment by m-lo** [comment]
image

**Comment by ericgmoon** [comment]
✅

**Review by ericgmoon — APPROVED** [review]
Not functional and no tests written for it yet (and we'll be switching to styled components eventually) - but approval for now as a WIP! Good work!!
